app/src/main/java/org/simple/clinic/user/UserSession.kt

```diff
@@ -53,12 +52,12 @@ import javax.inject.Named
 class UserSession @Inject constructor(
     private val loginApi: LoginApi,
     private val registrationApi: RegistrationApi,
-    private val findUserWithPhoneNumber: FindUserWithPhoneNumber,
+    private val refreshCurrentUser: RefreshCurrentUser,
```

Comment:
Not really sure with the direction we are moving towards, perhaps more context could help?

Reply:
Discussed offline: the purpose of this PR is to remove all network related code out of the `UserSession` in order to support the country specific endpoint. We will revisit the final design after doing a few more things:

- Breaking the DI graph into multiple scopes.
- Injecting the user directly into classes that require it instead of having them query it from the `UserSession`.